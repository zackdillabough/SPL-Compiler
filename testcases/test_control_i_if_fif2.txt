
a , b , c, d ;
{
input c;
input d;
c = c + d;
d = c + d;
IF d > c {
	output a;
	output b;
	output c;
	output d;
}
a = 1  ;
b = 2 ;
IF b > a {
	output a;
	output b;
	output c;
	output d;
}
	
input a;
input b;
 
IF  b > a {
        a = a+b;
	output a;
	IF a > b { 
		output a;
		b = a+b;
		IF b > a {
			output a;
			output b;
			IF a > b {
				output a;
				output b;
			}
		}
	}
	a = a+b;
	output a;
}
a = a*b;
output a;
output b;
IF a > b {
	output a;
	b = 1;
	a = a+b;
	output a;
	output b;
}

}
4 5 29 13 9
