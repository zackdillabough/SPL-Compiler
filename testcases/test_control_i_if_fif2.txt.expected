0 0 9 14 1 2 9 14 377 13 377 378 1 